spec/controllers/games_controller_spec.rb

```diff
@@ -44,6 +44,18 @@
     end
   end
 
+  describe "games#destroy" do
```

Comment:
basic test to make sure the destroy action is actually ending the game. I will add more conditions for this test (like checking if winner_id gets updated) once we get the Concede button working as intended.